devconf-outreachy/source/index.rst

```diff
@@ -0,0 +1,415 @@
+      OpenStack engineers, and standards organization, and
+    - taught Aija to negotiate intricate and complicated technical matters
+      with fellow engineers
+
+  The result proves her internship was successful.
+
+How to be a better mentor
+-------------------------
+
+Work-wise:
+
+* Keep your intern busy, challenged, and growing
+* Help keep them focused to get things done
+* Build on small successes, plan and promote small steps
+
+Community:
+
+* Network interns with the upstream community
+* Involve interns in code reviews
+
+Psychologically:
+
+* Understand the psychological setting, be aware of the biases
+* Consider being a co-researcher with your intern, rather than a
+  master-apprentice
+* Promote friendliness, awesomeness, and curiosity
+
+.. Things to talk about ^ (dtantsur)
+
+  Mentorship can be difficult, but rewarding.
+
+  You are likely to stay engaged with the intern on a daily basis,
+  making sure they are not stuck and remain challenged. This requires
+  time and dedication.
+
+  Considering the many moving parts in OpenStack, it is easy to get
+  distracted from one project to the other. This is also prompted
+  by long running reviews. Keeping the intern focused on their goals
+  is one of the things to focus on.
+
+  Good mentors should help interns to network with the community
+  through IRC chat and code reviews. Building community is the goal
+  here.
+
+  Being remote, it may be hard to read the intern's mind. We humans
+  are good at using our biases to build a substitute for
+  insufficient perception. That's a thing to stay away from.
+
+  No matter how experienced one may be, there will always be areas
+  in contemporary technology in which the mentors may not know much.
+  Therefore, sometimes we think of the mentorship as a research
+  collaboration, not so much as a benevolent master / submissive
+  apprentice kind of interaction.
+
+  After all, the internship should be a positive and fun experience.
+
+Signs of a successful intern
+----------------------------
+
+* Has a track record of playing with computers
+* Researched the project they applied to
+* Reasonable expectations, e.g., open source quirks
+
+.. Things to talk about ^ (ietingof)
+
+  It seems it's a promising sign when a person is genuinely interested in
+  the technology, not only as a prospective profession, but just for the
+  sake of it,
+
+  When they come well prepared understanding what the project is all
+  about and why it might be a good match for them, that's another good
+  sign. At best, they hope to stay with the project after the internship
+  is over.
+
+  Finally, one can't be too sensitive when it comes to the way the open
+  source community works. For instance, feedback can hurt and your work
+  could be abandoned or reshaped in a way you do not like. So, it's best
+  to see the applicant understanding the potential for such development.
+
+Come join us!
+-------------
+
+* If you are a student...
```

Comment:
Change to "If you are a prospective intern..."  Outreachy interns don't need to be students.

Reply:
Done